projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -6,39 +6,54 @@ import { useSelect } from '@wordpress/data';
 import { store as editorStore } from '@wordpress/editor';
 import { useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import { isModalSupportedByFlow } from './util';
 import './editor.scss';
 
 export const name = 'launchpad-save-modal';
 
 export const settings = {
 	render: function LaunchpadSaveModal() {
-		const isSaving = useSelect(
+		const isSavingSite = useSelect(
 			select => select( editorStore ).isSavingNonPostEntityChanges(),
 			[]
 		);
-		const prevIsSaving = usePrevious( isSaving );
+		const isSavingPost = useSelect( select => select( editorStore ).isSavingPost(), [] );
+
+		const prevIsSavingSite = usePrevious( isSavingSite );
```

Comment:
Would you mind elaborating on why we need the previous `isSaving` value?

Reply:
Because the selector is changing from `false` to `true` when you hit the publish button (when the saving is in progress) and back to `false`. I want to render the modal only after the post has been published (or the changes saved in the case of the site editor) -> That's why I check if the current value is `false` and the previous value was `true`. If I was to only check if the current value is `true` it would open the modal at the time of saving (still in progress)